junit-platform-launcher/src/main/java/org/junit/platform/launcher/listeners/OutputDir.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2015-2022 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v2.0 which
+ * accompanies this distribution and is available at
+ *
+ * https://www.eclipse.org/legal/epl-v20.html
+ */
+
+package org.junit.platform.launcher.listeners;
+
+import static org.apiguardian.api.API.Status.INTERNAL;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.SecureRandom;
+import java.util.Optional;
+import java.util.function.Supplier;
+
+import org.apiguardian.api.API;
+import org.junit.platform.commons.util.StringUtils;
+
+@API(status = INTERNAL, since = "1.8")
```

Comment:
Since `1.9`

Reply:
Fixed in 4c5320de38d4fe19e89dc1a3c13464743b573352.